fargate/README.md

```diff
@@ -39,19 +58,23 @@ You must run the Alert Logic Agent Container as a sidecar to each Fargate ECS ta
      ]
    }
    ```
-For the Cloud Defender platform only, in the task definition, update the ```value``` variable with your unique registration key. 
+In the task definition, update the ```value``` of the ```KEY``` environment variable with your unique registration key. 
+
+Alternatively, another option is to 'Create a New Revision' using the Builder and 'Add Container' with the above image. You must then add the environment variable ```KEY=<your_registration_key_here>```.
+
+2. Update the Service to use the latest Task Definition revision. 
+
+The service tasks will now be updated and the agent will show in the Health section of the Alert Logic portal in due course.
```

Comment:
Cloud Defender experience does not have the Health section. Instead, it has Hosts and Sources screens.